app/views/patients/_form.html.erb

```diff
@@ -47,75 +28,105 @@
                                    placeholder: '24 años',
                                    class: 'w-full rounded p-2 text-gray-900 border border-gray-300 focus:ring-blue-500 focus:border-blue-500' %>
         </div>
-      </div>
-
-      <div class = "w-full pt-2">
-        <%= f.label :reference, 'Referencia:' %>
-        <%= f.text_field :reference, placeholder: 'Nombre de quien referenció', class: 'w-full rounded p-2 text-gray-900 border border-gray-300 focus:ring-blue-500 focus:border-blue-500' %>
-      </div>
-
-      <div class = "w-full pt-2">
-        <%= f.label :occupations, 'Ocupaciones:' %>
-        <%= f.text_field :occupations, placeholder: 'Deporte, Danza, Lectura...',
-                                       class: 'w-full rounded p-2 text-gray-900 border border-gray-300 focus:ring-blue-500 focus:border-blue-500' %>
+        <div>
+          <%= f.label :gender, 'Sexo:' %>
+          <%= f.select :gender, options_for_select(gender_select_options, selected: @gender), {}, {class: 'w-full rounded p-2 text-gray-900 border border-gray-300 focus:ring-blue-500 focus:border-blue-500'} %>
+        </div>
+        <div>
+          <%= f.label :marital_status, 'Estado Civil:' %>
+          <%= f.select :marital_status, options_for_select(marital_status_select_options, selected: @marital_status), {}, {class: 'w-full rounded p-2 text-gray-900 border border-gray-300 focus:ring-blue-500 focus:border-blue-500'} %>  
```

Comment:
In all this selects you already have access to the patient instance, just ask for the value on each attribute
There's no need to create anda carry instance variables all over the template
```suggestion
          <%= f.select :marital_status, options_for_select(marital_status_select_options, selected: @patient.marital_status), {}, {class: 'w-full rounded p-2 text-gray-900 border border-gray-300 focus:ring-blue-500 focus:border-blue-500'} %>  
```